app.js

```diff
@@ -53,9 +33,13 @@ btnTScrollUp.addEventListener("click", function () {
 });
 
 btnScrollInstallation.addEventListener("click", function () {
-  var menuitem = document.getElementById("menu");
-  menuitem.style.display = "none";
-  myFunctiontoggle();
+  console.log("Ins");
+  console.log(screenWidth);
```

Comment:
always remove `console.logs` from final code

Reply:
Removed them. 
I Will keep this in mind.
Thanks.